spec/requests/accessible_format_request_spec.rb

```diff
@@ -0,0 +1,223 @@
+require "rails_helper"
+require "gds_api/test_helpers/publishing_api"
+
+RSpec.describe "Requests for accessible formats of documents", type: :request do
+  include GdsApi::TestHelpers::PublishingApi
+
+  let(:base_params) { { content_id: "123abc", attachment_id: "456def" } }
+  let(:base_param_string) { "content_id=123abc&attachment_id=456def" }
+  let(:content_id) { "123abc" }
+  let(:attachment_id) { "456def" }
+  let(:content_title) { "A document with some inaccessible attachments" }
+  let(:attachment_title) { "Inaccessible CSV" }
+  let(:base_path) { "/government/publications/example-document" }
+  let(:alternative_format_contact_email) { "format_request@example.com" }
+  let(:inaccessible_attachment) do
+    {
+      id: attachment_id,
+      url: "/government/publications/example-document/inaccessible-spreadsheet",
+      title: attachment_title,
+      accessible: false,
+      alternative_format_contact_email: alternative_format_contact_email,
+    }
+  end
```

Comment:
We have quite a lot of let blocks here that aren't actually needed. I believe lines 9 - 23 could be replaced by including the values directly into the content item hash (line 25) (content_id, attachment_id, inaccessible_attachment etc are just used in the setup). We could also get rid of base_path, alternative_format_contact_email etc as we could just copy the values to where they're used (one test) or dig into the content_item, e.g content_item[:alternative_format_contact_email], although this would be clunky for alternative_format_contact_email.

Reply:
Yeah, these were copied over from the old code, where IIRC they were needed (because it was a lot more complex). I'll go through and reduce them.